### PR TITLE
feat: implement products module

### PR DESCRIPTION
**Pull request by viniarruda**

### CODE REVIEW

File: src/products/products.service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      color: [
+        'red',
+        'grey',
+        'white',
+      ],
+      spotlight: false,
+    },
+  ];
+
+  getAllProducts() {
```
Comment (RafaelGSS) [inline src/products/products.service.ts]:
```suggestion
  getAllProducts(): Array {
```

File: src/products/products.service.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return this.products;
+  }
+
+  getProductsSpotlights() {
```
Comment (RafaelGSS) [inline src/products/products.service.ts]:
```suggestion
  getProductsSpotlights(): Array {
```

File: src/products/products.service.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      sku: 123456,
+      price: 189.55,
+      quantity: 10,
+      color: [
+        'red',
+        'grey',
+        'white',
+      ],
+      spotlight: false,
+    },
```
Comment (RafaelGSS) [inline src/products/products.service.ts]:
```suggestion
    }
```

File: src/products/products.service.ts (hunk 1 above)
Comment (viniarruda) [reply src/products/products.service.ts]: Yes, lint and prettier

**Pull request merged**